bot/exts/info/subscribe.py

```diff
@@ -0,0 +1,149 @@
+import arrow
+import discord
+from discord.ext import commands
+from discord.interactions import Interaction
+
+from bot import constants
+from bot.bot import Bot
+from bot.decorators import in_whitelist
+from bot.log import get_logger
+from bot.utils import checks, members, scheduling
+
+# Tuple of tuples, where each inner tuple is a role id and a month number.
+# The month number signifies what month the role should be assignable,
+# use None for the month number if it should always be active.
+ASSIGNABLE_ROLES = (
+    (constants.Roles.announcements, None),
+    (constants.Roles.pyweek_announcements, None),
+    (constants.Roles.advent_of_code, 12),
+    (constants.Roles.lovefest, 2),
```

Comment:
This does not have the same order as the constants file and default config 😅

It's fine to leave it but I have a strong urge to point it out

Reply:
https://github.com/python-discord/bot/pull/1868/commits/955160176fba2f524f88a6b95e8f9a7917f0bc29